Response-Errors.md

```diff
@@ -0,0 +1,123 @@
+| `NOT_FOUND_ERROR` | `3010` | Payment was not found | Searching payment does not exists | Getting payment |
+| `NOT_FOUND_ERROR` | `3011` | The searched payment document is not found | You will get this error if vendor's invoice functionality is disabled and you try to get an invoice | Preview invoice/receipt / send invoice |
+| `NOT_FOUND_ERROR` | `3012` | Vendor was not found  | Searching vendor does not exists | Getting vendor |
+| `NOT_FOUND_ERROR` | `3013` | User does not have such vendor | You will get this error if you try to create a payment for a shopper whose vendor is different than yours | Creating payment |
+| `NOT_FOUND_ERROR` | `3014` | Shopper was not found | Searching shopper does not exists | Getting shopper |
+| `NOT_FOUND_ERROR` | `3015` | API User was not found! | Searching api user does not exists | Getting api user |
+| `NOT_FOUND_ERROR` | `3016` | User was not found | Searching user does not exists | Getting user |
+| `NOT_FOUND_ERROR` | `3017` | Organization was not found! | Searching organization does not exists | Getting organization |
+| `NOT_FOUND_ERROR` | `3018` | Wallet was not found | You will get this error if you try to activate a wallet which your organization does not hold | Activating vendor wallet |
+| `NOT_FOUND_ERROR` | `3019` | The record you have searched for does not exists | ou will get this error if you pass invalid input, which we can not decode to determine what you are searching for | Every time we can not decode searching filter |
+
+
+### 4. Validation Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `VALIDATION_ERROR` | `4011` | Nonexistent country code | You will get this error if you pass non-existing country code to Vendor, CardHolderInformation | Processing Fiat payment / calculating VAT  |
+| `VALIDATION_ERROR` | `4111` | isCompany should be true/false | You will get this error if you pass non-boolean value to CardHolderInformation.isCompany | Processing payment |
+| `VALIDATION_ERROR` | `4211` | weiAmount must have non-zero value  | You will get this error if your `fundTx.weiAmount` is zero | Creating payment |
+| `VALIDATION_ERROR` | `4311` | Vat number (**your vat number**) is invalid | You will get this error if you pass invalid VAT number in CardHolderInformation | Creating vendor / Processing Fiat payment | 
+| `VALIDATION_ERROR` | `4312` | Vat number is empty | You will get this error if you have vat number as property in CardHolderInformation, but it does not have a value | Creating vendor / Processing Fiat payment |
+
+### 5. Global Errors
+
+| Error Name            | Code  | Message    | Description | Оccurrence |
+| --------------------- | ----- |--------------- | -------- | -------- |
+| `INTERNAL_SERVER_ERROR` | `5001` | Something went wrong. Contact your support for more information | You will get this error if something internally breaks down | -
```

Comment:
`You will get this error if something internally breaks down`
->
`General error`
I think that it is better not to say that the problem is internal or something broke down ;d